util/monitoring.js

```diff
@@ -0,0 +1,111 @@
+const _ = require('lodash');
+const googleMonitoring = require('@google-cloud/monitoring');
+const fs = require('fs');
+
+function getValueFromPoint(point) {
+  const valueKey = _.get(point, 'value.value');
+  return _.get(point, `value.${valueKey}`);
```

Comment:
Maybe we could use 0 as the default value? (the third argument?)

Reply:
Done!